src/components/inputs/text-area-input.tsx

```diff
@@ -0,0 +1,50 @@
+import { Text } from 'native-base';
+import React from 'react';
+import { TextInput, TextInputProps, TextStyle, View } from 'react-native';
+
+import { useTextAreaInputStyles } from './input.styles';
+
+interface TextAreaInputProps extends TextInputProps {
+  label?: string;
+  error?: string;
+  style?: TextStyle | TextStyle[];
+  testId?: string;
+  numberOfLines?: number;
```

Comment:
Except this, I expect all on TextinputProps

Reply:
I would expect this to derive from our own input component, and hence all other props should not exists.